File: java_backup/my java/Faceoff/Number.java
class Number
{
    public static void main(int n)
    {
        int n1=1,f,k,j,s=0;
        while(n>0)
        {
            k=n1%10;
            f=1;
            for(j=1;j<=k;j++)
            {
                f=f*j;
                s=s+f;
                n1=n/10;
            }
            if(s==n)
            System.out.println("krishnamurty number");
            
            else
            System.out.println("not a krishnamurty number");
        }
    }
}

File: 6th_sem/MATLABCIA/MATLAB_ANUBHAV/number.m
clc;
clear;
element = zeros (1,50);
ctr=1;
i=1;
y=1;
freq=1;
    num = input('Please state how many numbers you want to input: ');
    while (ctr <= num) %input number in array
     element (ctr) = input( 'Enter number: ');
     ctr = ctr+1;
    end
    for ctr=1:1:num  %to arrange number in descending
     for  i=1:1:num
            if element (i) < element (i+1) 
                     temp = element(i);
                     element (i) = element (i+1);
                     element (i+1) = temp; 
             end
     end
    end
disp('N Count');
while (y<=num) %counting frequency of number
k = element(y);
freq=1;
    if k == element (y+1) 
             for x = y:num
                    if k == element (x+1) 
                     freq = freq+1;
                    end
             end
             fprintf('%d\t%d\n',k, freq);
    else
        fprintf('%d\t%d\n',k, freq);
    end
    y=y+freq;
end